Task 2.1.txt

```diff
@@ -0,0 +1 @@
+x=mean, y=standart deviation

```

Comment:
Maybe it should be "standard" not "standart". Otherwise looks good!

Reply:
I agree!